docs/code/os-dev.md

```diff
@@ -78,15 +78,17 @@ sudo apt install appname --reinstall
 
 ## WingPanel {#wingpanel}
 
-When developing the Panel (codenamed WingPanel) or panel-related packages like the Applications Menu and indicators, you want to start WingPanel from the command line to view logs. WingPanel is automatically started and restarted by Cerbere. To remove WingPanel from its monitored applications:
+When developing the Panel (codenamed WingPanel) or panel-related packages like the Applications Menu and indicators, you want to start WingPanel from the command line to view logs. WingPanel is automatically started and restarted by the GNOME session manager "gnome-session". To prevent WingPanel from automatically restarting kill it twice in a row:
 
-1. In dconf Editor, browse to `/io/elementary/desktop/cerbere/monitored-processes`
-2. Disable "Use default value"
-3. Change the "Custom value" to `['plank']`
-4. In Terminal run `killall wingpanel` to stop the current WingPanel
-5. Start wingpanel by running `wingpanel`
+```bash
+killall wingpanel && killall wingpanel
```

Comment:
afaict, this doesn't actually work. I don't think there's enough time here for the panel to have been started again

Reply:
Yes, it didn't work when I tried it now. I guess, it worked when I initially tested it, because I already killed Wingpanel before. I've added a  `sleep 1` to give the panel time to restart.